composites/Plugin/ContentAnalysis/actions/tests/ContentAnalysisTest.js

```diff
@@ -128,3 +132,53 @@ describe( "removeKeyword function", function() {
 		expect( actual ).toEqual( expected );
 	} );
 } );
+
+describe( "setOverallScore readability function", function() {
+	it( "creates the setOverallScore for readability results action", function() {
+		let results = [
+			{ id: "result", score: 3, description: "This is a bad score!", markingIsActive: false },
+			{ id: "overallScore", score: 3 },
+		];
+
+		const expected = {
+			type: SET_OVERALL_SCORE_READABILITY,
+			results: results,
+		};
+		const actual = setOverallScoreReadability( results );
+		expect( actual ).toEqual( expected );
+	} );
+} );
+
+describe( "setOverallScore seo function", function() {
+	it( "creates the setOverallScore for seo results action", function() {
+		let keyword1 = "keyword1";
```

Comment:
Seems unnecessary to explicitly assign this variable; you can hard-code the string in the mock scorePerKeyword object.